src/main/resources/mixins.common.realtime.json

```diff
@@ -0,0 +1,28 @@
+{
+  "minVersion": "0.5.1",
+  "package": "org.spongepowered.common.mixin.realtime.mixin",
```

Comment:
I needed to put the mixins in a different package than the interface. It was either this or make an "interface" package.